CMakeLists.txt

```diff
@@ -351,6 +362,112 @@ if(UA_ENABLE_MULTITHREADING)
     MESSAGE(WARNING "UA_ENABLE_MULTITHREADING is enabled. The feature is under development and marked as EXPERIMENTAL")
 endif()
 
+############################
```

Comment:
Can we put this whole linting section into a subfolder and just include it here. e.g. into the tools/CMakeLists.txt

This file is already quite long.

Reply:
Yes, but I think the division into separate include files should be done in a different PR,
because it includes several other things too.